guides/App Versions.md

```diff
@@ -21,7 +21,7 @@ The changes made to your app are reviewed and the new version is classified as e
 ### Major Updates
 The following changes to your app require a major update. These are:
 - Adding or removing [Permissions](https://devforum.wix.com/kb/en/article/about-permissions)
-- Adding an [Embedded Script](https://devforum.wix.com/kb/en/article/set-up-an-embedded-script-component) component
+- Adding an [Embedded Script](https://dev.wix.com/docs/build-apps/developer-tools/extensions/embedded-scripts) extension
```

Comment:
Is this your change? If not, is the updated link correct?

Reply:
Yes, the old link redirects to the new link.